core/templates/domain/utilities/classifier-file.model.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright 2017 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Model class for classifier data files.
```

Comment:
I thought of keeping this together with audio-file.model.ts and image-file.model.ts because all three of them are essentially binary files fetched from backend / GCS (in prod).

Reply:
Done